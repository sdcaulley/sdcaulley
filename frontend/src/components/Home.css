.home-nav {
  width: 90%;
}

.home-nav__ul {
  list-style: none;
  width: 90%;
}

.home-nav__ul_flex-container {
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.home-nav__ul__li {
  border-color: #7a1f1f;
  border-radius: 1em;
  border-style: solid;
  border-width: medium;
  box-shadow: 10px 10px 5px tan;
  display: block;
  margin: 1em;
  padding: 0.5em;
  text-align: center;
  font-size: 2em;
}

.home-nav__Link {
  color: brown;
  text-decoration: none;
}

@media only screen and (min-width: 640px) {
  .home-nav__ul {
    margin: 10em auto;
  }
  .home-nav__ul_flex-container {
    flex-direction: row;
    justify-content: space-around;
  }

  .home-nav__ul__li {
    height: 3em;
    margin: 0;
    padding: 0;
    width: 8em;
  }

  .home-nav__Link {
    display: block;
    margin: auto;
    padding: 0.9em;
  }
}
