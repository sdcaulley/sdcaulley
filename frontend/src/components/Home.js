import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Home extends Component {
    render() {
        return (
            <nav>
            <ul>
                <li>
                    <Link to='/code'>Code</Link>
                </li>
                <li>
                    <Link to='/consciousness'>Consciouness</Link>
                </li>
                <li>
                    <Link to='/craft'>Craft</Link>
                </li>
            </ul>
            </nav>
        );
    }
}

export default Home;
