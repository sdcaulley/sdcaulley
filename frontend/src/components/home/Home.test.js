import React from 'react';
import Home from './Home';
import { create } from 'react-test-renderer';

describe('Home', () => {
    test('should match snapshot', () => {
        const component = create(<Home />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
