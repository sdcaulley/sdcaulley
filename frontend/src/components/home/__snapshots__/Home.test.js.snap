// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot 1`] = `
<nav
  className="home-nav"
>
  <ul
    className="home-nav__ul home-nav__ul_flex-container"
  >
    <li
      className="home-nav__ul__li home-nav__ul__li_flex-item"
    >
      <a
        className="home-nav__Link"
        href="/code"
        onClick={[Function]}
      >
        Code
      </a>
    </li>
    <li
      className="home-nav__ul__li home-nav__ul__li_flex-item"
    >
      <a
        className="home-nav__Link"
        href="/consciousness"
        onClick={[Function]}
      >
        Consciousness
      </a>
    </li>
    <li
      className="home-nav__ul__li home-nav__ul__li_flex-item"
    >
      <a
        className="home-nav__Link"
        href="/craft"
        onClick={[Function]}
      >
        Craft
      </a>
    </li>
  </ul>
</nav>
`;
